resources/lang/vendor/hearth/fr/auth.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Authentication Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are used during authentication for various
+    | messages that we need to display to the user. You are free to modify
+    | these language lines according to your application's requirements.
+    |
+    */
+
+    'sign_in' => 'S’identifier',
+    'sign_out' => 'Se déconnecter',
+    'create_account' => 'Créer un compte',
+    'label_password' => 'Mot de passe',
+    'label_password_confirmation' => 'Confirmez le mot de passe',
+    'existing_account_prompt' => 'Vous avez déjà un compte?',
+    'create_your_account' => 'Créez votre compte',
+    'label_current_password' => 'Mot de passe actuel',
+    'change_password' => 'Changer le mot de passe',
+    'action_cancel' => 'Annuler',
+    'action_confirm' => 'Confirmer',
+    'password_change_succeeded' => 'Votre mot de passe a été modifié.',
+    'label_remember_me' => 'Souvenez-vous de moi',
+    'error_intro' => 'Désolé, quelque chose s’est mal passé.',
+    'wrong_password' => 'Le mot de passe fourni est incorrect.',
+    'confirm_intro' => 'Pour votre sécurité, veuillez confirmer votre mot de passe avant de continuer.',
+    'forget_prompt' => 'Mot de passe oublié?',
+    'forgot_intro' => 'Mot de passe oublié? Entrez votre adresse e-mail et nous vous enverrons par e-mail un lien de réinitialisation du mot de passe qui vous permettra d’en choisir un nouveau.',
+    'forgot_submit' => 'Envoyer le lien de réinitialisation',
+    'reset_submit' => 'Réinitialiser le mot de passe',
+    'verification_required' => 'Vérification de l\'e-mail requise',
```

Comment:
I see that we escaped a single quote here, but could we make the same update we did for the single quote in Hearth as well?

Reply:
Yes, I guess we could do that.